pkg/synopkg/PeerBanHelperPackage/INFO.sh

```
@@ -8,13 +8,14 @@ version="@PBH_VERSION@"
 displayname="PeerBanHelper"
 os_min_ver="7.2-64555"
 maintainer="PBH-BTN"
+maintainer_url="https://github.com/PBH-BTN/PeerBanHelper"
```

Shellcheck

[warning] maintainer_url appears unused. Verify use (or export if used externally).

(SC2034)